### PR TITLE
docs(halyard/ha): Rename echo-slave to recho-replicax.

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

**Comment by joonlim** [comment]
@jtk54 @lwander Renaming 'echo-slave' to 'echo-replica'

**Review by lwander — APPROVED** [review]

File: reference/halyard/high-availability.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ hal config deploy ha echo enable
 When Spinnaker is deployed with this enabled, Echo will be deploy as two different services:
 
 * [`echo-scheduler`](#echo-scheduler)
-* [`echo-slave`](#echo-slave)
+* [`echo-replica`](#echo-replica)
 
 Although the logical Echo services still cannot be horizontally scaled, splitting the services will reduce the load on both.
```
Comment (duftler) [inline reference/halyard/high-availability.md:108]: One of them can be horizontally scaled, right?

Comment (joonlim) [reply reference/halyard/high-availability.md:108]: Thanks for catching that. I'll make an update to this now.

Comment (joonlim) [reply reference/halyard/high-availability.md:108]: Fixed

**Pull request merged**